lib/acpi/acpi_shell.c

```diff
@@ -104,7 +104,14 @@ static void dump_dev_res(const struct shell *sh, ACPI_RESOURCE *res_lst)
 			shell_print(sh, "ACPI_RESOURCE_TYPE_EXTENDED_ADDRESS64");
 			break;
 		case ACPI_RESOURCE_TYPE_EXTENDED_IRQ:
+			ACPI_RESOURCE_EXTENDED_IRQ * ext_irq_res = &res->Data.ExtendedIrq;
```

Comment:
Incorrect style: there should be no space between `*` and the variable name. There's a false-positive Compliance error because of this, which you should solve the same way as some other already existing case statements do, i.e. add a `{` at the end of the case statement.